core/src/banking_stage/consumer.rs

```diff
@@ -46,21 +48,16 @@ pub const TARGET_NUM_TRANSACTIONS_PER_BATCH: usize = 64;
 
 pub struct ProcessTransactionBatchOutput {
     // The number of transactions filtered out by the cost model
-    pub(crate) cost_model_throttled_transactions_count: usize,
```

Comment:
noticed this in a few places. Why use u64 for counts over usize?

Reply:
Because they are all eventually converted to u64 for metrics reporting or for updating bank stats